span-processing-config-service-api/src/main/proto/org/hypertrace/span/processing/config/service/v1/span_processing_config_service.proto

```diff
@@ -149,3 +150,46 @@ message ApiNamingRuleMetadata {
   google.protobuf.Timestamp creation_timestamp = 1;
   google.protobuf.Timestamp last_updated_timestamp = 2;
 }
+
+message SamplingConfig {
+  string id = 1;
+  Scope scope = 2;
+  RateLimitConfig rate_limit_config = 3;
+  SpanFilter filter = 4;
+  bool disabled = 5;
+}
+
+enum Scope {
+  SCOPE_UNSPECIFIED = 0;
+  SCOPE_API = 1;
+  SCOPE_ENVIRONMENT = 2;
+  SCOPE_TENANT = 3;// if no sampling config,use default
+}
+
+message RateLimitConfig {
+  RateLimit trace_limit_per_endpoint = 1;
+  RateLimit trace_limit_global = 2;
+  google.protobuf.Duration api_endpoint_cache_duration = 3;
+}
+
+message RateLimit {
+  oneof limit {
+    WindowedRateLimit fixed_window_limit = 1;
+  }
+}
+
+message WindowedRateLimit {
+  int64 quantity_allowed = 1;
+  google.protobuf.Duration window_duration = 2;
+}
+
+message IncludeSpanRule {
```

Comment:
IncludeSpanRules currently look very similar to exclude span rules. Instead of `SpanFilter`, we could use something similar to the below as they would most likely be configured based on URL regex. But used `SpanFilter` as it is an umbrella over the below one and provides more flexibility.
```
message UrlRegexInfo {
   string regex = 1;
   string value = 2;
}
```